tests/pipelines/stable_diffusion/test_stable_diffusion.py

```diff
@@ -585,14 +585,7 @@ def test_stable_diffusion_long_prompt(self):
         negative_prompt = None
         num_images_per_prompt = 1
         logger = logging.get_logger("diffusers.pipelines.stable_diffusion.pipeline_stable_diffusion")
-
-        prompt = 25 * "@"
-        with CaptureLogger(logger) as cap_logger_3:
-            negative_text_embeddings_3, text_embeddings_3 = sd_pipe.encode_prompt(
-                prompt, torch_device, num_images_per_prompt, do_classifier_free_guidance, negative_prompt
-            )
-            if negative_text_embeddings_3 is not None:
-                text_embeddings_3 = torch.cat([negative_text_embeddings_3, text_embeddings_3])
+        logger.setLevel(logging.WARNING)
```

Comment:
@DN6 we set the logger to different levels for different tests. The problem is that we often don't change the level back of the logging and therefore there can be a mismatch between running a test in isolation and running all tests. 

In this case, I was able to reproduce the bug when running all tests of stable diffusion in contrast to just running the single test (in case you can't reproduce a bug locally, it's always a good idea to just run all fast tests in which case the bug can be reproduced most of the time).

Here this "flaky" test can be fixed by forcing the logger level to the correct one before.

Reply:
Also this bug was not introduced by your PR, it was more just trigger because of just changes I think. Anyways always a good idea to try to find out what the problem is and to just fix it then

Reply:
Nice! Thanks for explaining and fixing 🙏🏽